Add Instructions on how to get Dagger Engine to work on RHEL to the Docs (#6996)
dagger/dagger: docs/current_docs/manuals/user/932727-troubleshooting.mdx

DAGGER_ENGINE_DOCKER_CONTAINER="$(docker container list --all --filter 'name=^dagger-engine-*' --format '{{.Names}}')"
docker restart "$DAGGER_ENGINE_DOCKER_CONTAINER"
```

> ## Dagger Engine Restarts Due to `CNI setup error`

Reviewer: ```suggestion
## Dagger restarts with a "CNI setup error"
```

> Dagger Engine requires the `iptable_nat` Linux Kernel Module in order to function properly. On some Linux Distributions (including RHEL 8.x and 9.x) this module is not loaded by default.

Reviewer: ```suggestion
The Dagger Engine requires the `iptable_nat` Linux kernel module in order to function properly. On some Linux distributions (including Red Hat Enterprise Linux 8.x and 9.x), this module is not loaded by default.
```

You can load this module by running `sudo modprobe iptable_nat`.

> If you'd like to have this module loaded by default when your machine starts, then you can add it to the `/etc/modules-load.d/modules` file with the following command:

Reviewer: ```suggestion
To have this module loaded automatically on startup, add it to the `/etc/modules-load.d/modules` file with the following command:
```
